document/PolicyComplianceProfile.mdk

```diff
@@ -1 +1,65 @@
-# Policy Compliance Profile { #policy-compliance-profile }
\ No newline at end of file
+# Policy Compliance Profile { #policy-compliance-profile }
```

Comment:
We might be able to dispense with that if the Policy Compliance Profile changes `runInfo.analysisTargets` from optional ("may contain") to required ("shall contain"). @michaelcfanning?